imagesloaded/index.d.ts

```diff
@@ -1,9 +1,9 @@
-// Type definitions for imagesLoaded 3.1.8
+// Type definitions for imagesLoaded 4.1.1
```

Comment:
For future references, the version should be just `major.minor`, the build is ignored. Adding tslint.json would have flagged this, see https://github.com/DefinitelyTyped/DefinitelyTyped#lint for more details.